src/kirby/components/slide-button/slide-button.component.scss

```diff
@@ -24,35 +26,35 @@
     width: 100%;
   }
 
-  .range-container {
-    $padding: 2px;
-
+  $slide-button-padding: 2px;
+  $slider-button-container-radius: 28px;
```

Comment:
Magic number - should be calculated

Reply:
Calculated based on what? Can you be more specific?

Reply:
Calculated as half of height

Reply:
Done